scripts/install_ddev_wsl2_docker_desktop.ps1

```diff
@@ -41,8 +34,41 @@ if (-not(wsl -e docker ps) ) {
     throw "Docker Desktop integration with the default distro does not seem to be enabled yet."
 }
 $ErrorActionPreference = "Stop"
-# Install needed choco items
-choco upgrade -y ddev gsudo mkcert
+
+# Install DDEV on Windows to manipulate the local hosts file.  (Also requires mkcert & sudo; see below.)
+$TempDir = $env:TEMP
+$DdevInstallerPath = Join-Path $TempDir "ddev-installer.exe"
+# TODO: To always fetch the latest EXE (e.g. https://github.com/<OWNER>/<REPO>/releases/latest/download/myprogram.exe),
+# there can't be version numbers in the file name so we need to remove the version number from the release artefact.
+# Until then, we can simply fetch the installer from the latest release, which we'll hardcode.
+Invoke-WebRequest `
+    -Uri "https://github.com/ddev/ddev/releases/download/v1.24.3/ddev_windows_amd64_installer.v1.24.3.exe" `
```

Comment:
You'll find a number of examples of getting latest version in our Dockerfile and in the [install_ddev.sh](https://github.com/ddev/ddev/blob/main/scripts/install_ddev.sh)

Reply:
Done!